### DISCUSSION TITLE
Hi!

### DISCUSSION THREAD

**Discussion by fatheroftodds** [evennia/evennia#3683] [Greetings & Introductions]
Hi everyone, I'm Liam aka Vermidius. My MUD experience has mostly been through IRE games, which I enjoyed for the majority of my spare time (and sadly some spare cash) for around 10 years. I've always been intrigued by the idea of making my own MUD as a hobbyist and have some ideas in mind for a Scifi MUD based loosely on a few book series' that I enjoy. 

I'm working my way through the Evennia tutorials and really enjoying this journey so far!

**Comment by Griatch** [comment]
Hi and welcome @fatheroftodds. Sci-fi MUD sounds fun. Good luck with your learning and don't be shy to ask (in the discussion forum or on discord) if you have any questions :)
